index.js

```diff
@@ -10,40 +10,41 @@ import {
     PointPropType,
     findNodeHandle,
 } from 'react-native';
-var RNTableViewConsts = NativeModules.UIManager.RNTableView.Constants;
 var resolveAssetSource = require('react-native/Libraries/Image/resolveAssetSource');
 
+var RNTableViewConsts = NativeModules.RNTableViewManager.Constants;
```

Comment:
Rather than extract the constants directly off the `RNTableViewManager` I would have thought you'd just import the `UIManager` directly from react-native as in the example [here](https://facebook.github.io/react-native/docs/native-components-ios.html#styles).

So, the code would be something like:
```
import {
    ReactNativeViewAttributes,
    ...
    UIManager,
} from 'react-native';
var RNTableViewConsts = UIManager.RNTableView.Constants;
...
```

Reply:
I was confused about it too, but I think it was a change in `0.49`. https://facebook.github.io/react-native/docs/native-modules-ios.html#exporting-constants shows the way I did it here. The reason I changed it is that `UIManager.RNTableView.Constants` was undefined, and caused the app the crash